images/prow-tests/Makefile

```diff
@@ -15,13 +15,7 @@
 IMAGE_NAME = prow-tests
 include ../simple-image.mk
 
-push_coverage_dev: build
-	docker tag $(IMG):$(TAG) $(IMG):coverage-dev
-	docker push $(IMG):coverage-dev
-
-push_beta: confirm-master build
-	docker tag $(IMG):$(TAG) $(IMG):beta
-	docker push $(IMG):beta
-
-push:: push_beta
-
+cloud_build:
+	gcloud builds submit --config=./cloudbuild.yaml --substitutions=_TAG=$(TAG),_IMG=$(IMG) \
+		--machine-type=n1-highcpu-32 \
```

Comment:
wow really?

Reply:
machine type?

Reply:
Yeah, that's quite a machine 😅

Reply:
We actually do not need such a large machine, I have changed it to `n1-highcpu-8`